Automattic/jetpack
# Featured content: Make the customizer extension only be shown if current theme is not a block theme

Fixes partially #31725

## Proposed changes:

* Check if current active theme is a block theme before attempting to add a section to the customizer

### Other information:


## Jetpack product discussion
https://github.com/Automattic/wp-calypso/issues/71130#issuecomment-1617982587

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
* Checkout this branch
* If you're using a modern theme expect to see no Customizer menu item under Appearance
